bike-erp/backend/src/main/services/accountingService/AccountingService.ts

```diff
@@ -1,115 +1,149 @@
-import { AccountPayableDAO } from "../../dao/AccountPayableDAO"
+import { AccountPayableDAO } from "../../dao/AccountPayableDAO";
 import { AccountReceivableDAO } from "../../dao/AccountReceivableDAO";
 import { BikeOrder } from "../../models/interfaces/BikeOrder";
 
 // This class handles all finance related features such as account receivable and payable
 export class AccountingService {
-
-    private static accountingService: AccountingService | undefined;
-    private static accountPayableDAO: AccountPayableDAO = new AccountPayableDAO()
-    private static accountReceivableDAO: AccountReceivableDAO = new AccountReceivableDAO();
-
-    private constructor() {
-    }
-
-    // Instanciating singleton
-    public static getAccountingService() {
-        if (this.accountingService === undefined) {
-            this.accountingService = new AccountingService();
-        } else {
-            return this.accountingService;
-        }
+  private static accountingService: AccountingService | undefined;
+  private static accountPayableDAO: AccountPayableDAO = new AccountPayableDAO();
+  private static accountReceivableDAO: AccountReceivableDAO = new AccountReceivableDAO();
+
+  private constructor() {}
+
+  // Instanciating singleton
+  public static getAccountingService() {
+    if (this.accountingService === undefined) {
+      this.accountingService = new AccountingService();
+    } else {
+      return this.accountingService;
     }
-
-    public static getAccountPayableDAO() {
-        return AccountingService.accountPayableDAO;
+  }
+
+  public static getAccountPayableDAO() {
+    return AccountingService.accountPayableDAO;
+  }
+
+  public static getAccountReceivableDAO() {
+    return AccountingService.accountReceivableDAO;
+  }
+  // Method for handling the creating of a account payable
+  public static async createAccountPayable(
+    orderList: Array<Order>,
+    email: string
+  ) {
+    let total = 0;
+
+    // Calculate total
+    orderList.forEach((order) => {
+      total = total + order.price * order.quantity;
+    });
+
+    try {
+      // Creating account payable in db
+      const accountPayableId = (await AccountingService.accountPayableDAO?.createAccountPayable(
+        total,
+        new Date().toISOString().slice(0, 19).replace("T", " "),
+        email
+      )) as number;
+      // Create transaction item in db for each item in order list
+      orderList.forEach(async (order) => {
+        const transactionItemId = (await AccountingService.accountPayableDAO?.createTransactionItems(
```

Comment:
Could you look into your formatter, cause it's creating a lot of line for small arguments

Reply:
I will change my formatter for future commits. Reverting them back would take a lot of time since the formatter automatically formats the whole file.